.github/workflows/codeql.yml

```diff
@@ -0,0 +1,76 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
```

Comment:
Was this file auto-generated, right? I think we don't need this NOTE here....

Reply:
Fixed